src/OrchardCore.Modules/OrchardCore.Flows/wwwroot/Scripts/flows.edit.js

```diff
@@ -58,7 +58,7 @@ $(function () {
     $(document).on('click', '.widget-delete', function () {
         var title = $(this).data('title');
         var message = $(this).data('message');
-        confirmDialog(title, message, r => {
+        confirmDialog(title, message, undefined, undefined, undefined, undefined, r => {
```

Comment:
Maybe an improvement would be to pass a settings object to the `confirmDialog`. function
```
var options = {title: 'title', message: 'message', callback: function(x){} ...};
confirmDialog(options);
```

Reply:
Perhaps I was running on a slow internet connection, if you look closely I just created a new branch few minutes ago, to improve this by using optional params instead

Reply:
Make sure to search and fix all calls of the confirmDialog() function

Reply:
Sure, they are in three modules only